test/e2e/app-dir/io-is-dynamic-ppr/app/cases/dynamic_api_search_params_server_boundary/page.tsx

```diff
@@ -0,0 +1,50 @@
+import { Suspense } from 'react'
+
+// @TODO the current implementation of an RSC render begins rendering pages before higher up
```

Comment:
this has been fixed in https://github.com/vercel/next.js/pull/68847 so we should be able to update this